contracts/plugins/assets/compoundv3/CusdcV3Wrapper.sol

```diff
@@ -286,9 +286,7 @@ contract CusdcV3Wrapper is ICusdcV3Wrapper, WrappedERC20, CometHelpers {
         (, uint64 trackingSupplyIndex) = getSupplyIndices();
         uint256 indexDelta = uint256(trackingSupplyIndex - baseTrackingIndex[account]);
 
-        baseTrackingAccrued[account] += safe64(
-            (safe104(accountBal) * indexDelta) / TRACKING_INDEX_SCALE
-        );
+        baseTrackingAccrued[account] += (safe104(accountBal) * indexDelta) / TRACKING_INDEX_SCALE;
```

Comment:
`houldnt we also change the `getRewardsOwed()` in L 259?

Reply:
Agreed.